### PR TITLE
Document query metadata

### PR DESCRIPTION
**Pull request by vsavkin**

### CODE REVIEW

File: modules/@angular/core/src/metadata/di.ts [hunk 1]
```diff
@@ -187,7 +174,45 @@ export interface ContentChildrenDecorator {
...
+ *   }
+ * }
+ * ```
+ *
+ * @description
+ *
+ * You can use ContentChildren to get the {@link QueryList} of elements or directives from the
+ * content DOM. Any time a child element is added, removed, or moved, the query list will be
+ * updated,
+ * and the changes observable of the query list will emit a new value.
```
Comment (kara) [inline modules/@angular/core/src/metadata/di.ts]: gulp formatting issue here?

File: modules/@angular/core/src/metadata/di.ts [hunk 2]
```diff
@@ -242,12 +242,50 @@ export interface ContentChildDecorator {
...
+ * import {Directive, ContentChild} from '@angular/core';
+ *
+ * @Directive({
+ *   selector: 'someDir'
+ * })
+ * class SomeDir {
+ *   @ContentChild(ChildDirective) contentChild;
+ *
+ *   ngAfterContentInit() {
+ *     // contentChildren is set
```
Comment (kara) [inline modules/@angular/core/src/metadata/di.ts]: This should be `contentChild`, no?

File: modules/@angular/core/src/metadata/di.ts [hunk 3]
```diff
@@ -380,92 +382,63 @@ export const ViewChildren: ViewChildrenDecorator = makePropDecorator(
...
+ *   templateUrl: 'someCmp.html'
  * })
  * class SomeCmp {
- *   @ViewChild(ChildCmp) child:ChildCmp;
+ *   @ViewChild(ChildDirective) child: ChildDirective;
  *
  *   ngAfterViewInit() {
- *     // child is set
- *     this.child.doSomething();
+ *     // viewChildren is set
```
Comment (kara) [inline modules/@angular/core/src/metadata/di.ts]: child or viewChild

**Comment by kara** [comment]
LGTM

**Comment by evmar** [comment]
rebased as 7ac47acc1c4b735f1237fa747db969f86833be3d

**Pull request closed without merging**